src/cli.ts

```diff
@@ -17,6 +17,10 @@ const main = defineCommand({
       description: "The directory to build",
       required: false,
     },
+    config: {
+      type: "string",
+      description: "The configuration file to use (defaults to ./build.config)",
+    },
```

Comment:
should add `required: false`

Reply:
args (non positional) are non required by default